dataset-spec/README.md

```diff
@@ -0,0 +1,86 @@
+| links           | [Link Object]                         | Links (required)                | A list of references to other documents, see Link Object for further documentation. TODO: Remove if catalog is revised. |
+
+### Provider Object
+
+| Element | Type   | Name                  | Description                                     |
+| ------- | ------ | --------------------- | ----------------------------------------------- |
+| name    | string | Organization name     | The name of the organization or the individual. |
+| url     | string | Organization homepage | Homepage of the provider.                       |
+
+###  Host Object
+
+| Element        | Type    | Name                  | Description                                                  |
+| -------------- | ------- | --------------------- | ------------------------------------------------------------ |
+| description    | string  | Description           | Detailed description to explain the hosting details. [CommonMark 0.28](http://commonmark.org/) syntax MAY be used for rich text representation. |
+| scheme         | string  | Scheme (required)     | Values: S3, GCS, URL, OTHER                                  |
+| id             | string  | Identifier (required) | Host-specific identifier such as an URL or asset id.         |
+| region         | string  | Region                | Provider specific region where the data is stored.           |
+| requester_pays | boolean | Requester pays        | `true` if requester pays, `false` if host pays.              |
+
+**Note:** The idea of storage profiles is currently [discussed](https://github.com/radiantearth/stac-spec/issues/148). Therefore, scheme, id and region may be removed from the final spec.
+
+### Link Object
+
+TODO: Should be compatible with STAC or remove if catalog is revised.
+
+| Element | Type   | Name                           | Description                         |
+| ------- | ------ | ------------------------------ | ----------------------------------- |
+| href    | string | Hyperlink reference (required) |                                     |
+| rel     | string | Relation (required)            |                                     |
+| type    | string | MIME-type                      | MIME-type of the referenced entity. |
+| title   | string | Title                          | Human-readable title for the link.  |
+
+##  EO extension (eo) - Items and Datasets
+
+We follow the STAC EO extension where meaningful (see below), but propose additional fields:
+
+| Element      | Type     | Name               | Description           |
+| ------------ | -------- | ------------------ | --------------------- |
+| asset_schema | object   | Asset Schema       | TODO                  |
+| nodata       | [number] | Nodata values      | The no data value(s). |
+| pyramid      | object   | Pyramid parameters | TODO                  |
```

Comment:
So far I only used this as en enum describing the way averaging is down during pyramiding: MEAN for normal continuous bands like EO observations or temperature/DEM/etc, MODE for categorical values (eg, landcover classes), SAMPLE for the rest (typically this means bitmask bands). SAMPLE means just take one possibly random value, as nothing else makes sense. (In reality in this case we always take UL value for each 2x2 grid).

Reply:
@simonff, could you come up with a proposal for this field? And maybe also asset_schema? These initially were your ideas and I think you have a much better idea to define them.